### PR TITLE
Set motion magic hard coded values to values in preferences

### PR DESCRIPTION
**Pull request by dunep**

### CODE REVIEW

File: src/main/java/frc/robot/RobotPreferences.java [hunk 1]
```diff
@@ -190,6 +190,11 @@ public static final class prefPitch {
     public static final SN_DoublePreference pitchP = new SN_DoublePreference("pitchP", 300); // 100
     public static final SN_DoublePreference pitchI = new SN_DoublePreference("pitchI", 0);
     public static final SN_DoublePreference pitchD = new SN_DoublePreference("pitchD", 0);
+    public static final SN_DoublePreference MotionMagicCruiseVelocity = new SN_DoublePreference(
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/RobotPreferences.java]: // - Motion Magic -
    /**
     * <b> Units: </b> Rotations per second (rps)
     */
    public static final SN_DoublePreference climberCruiseVelocity = new SN_DoublePreference(
        "climberCruiseVelocity", 80);

    /**
     * <b> Units: </b> Rotations per second per second (rps/s)
     */
    public static final SN_DoublePreference climberAcceleration = new SN_DoublePreference(
        "climberAcceleration", 160);
    /**
     * <b> Units: </b> Rotations per second per second per second (rps/s/s)
     */
    public static final SN_DoublePreference climberJerk = new SN_DoublePreference("climberJerk", 1600);

File: src/main/java/frc/robot/RobotPreferences.java [hunk 2]
```diff
@@ -305,10 +320,10 @@ public static final class prefShooter {
 
     public static final SN_DoublePreference leftShooterIntakeVelocity = new SN_DoublePreference(
         "leftShooterIntakeVelocity",
-        -10);
+        -3);
```
Comment (AngelaKimmy) [inline src/main/java/frc/robot/RobotPreferences.java]: Why were these values changed for this PR?

File: src/main/java/frc/robot/RobotPreferences.java [hunk 3]
```diff
@@ -454,6 +469,21 @@ public static final class prefTurret {
     public static final SN_DoublePreference turretCurrentTimeThreshold = new SN_DoublePreference(
         "turretCurrentTimeThreshold", 0.1);
 
+    /**
+     * <b> Units: </b> Rotations per second per second (rps)
+     */
+    public static final SN_DoublePreference turretCruiseVelocity = new SN_DoublePreference(
```
Comment (AngelaKimmy) [inline src/main/java/frc/robot/RobotPreferences.java:475]: Is this the correct value?

**Comment by alicekuznetsov** [comment]
Stale :(

**Pull request closed without merging**